src/protean/impl/repository/dict_repo.py

```diff
@@ -60,21 +61,15 @@ def _filter_objects(self, page: int = 1, per_page: int = 10,
 
             # Add objects that match the given filters
             for fk, fv in filters.items():
-                if type(fv) in (list, tuple):
-                    if item[fk] not in fv:
-                        match = False
-                else:
-                    if item[fk] != fv:
-                        match = False
+                lookup_class = self.get_lookup(fv[0])
+                lookup = lookup_class(item[fk], fv[1])
+                match &= eval(lookup.as_expression())
```

Comment:
Do we have a way other than `eval`? 
btw, what is `&=` ?

Reply:
`&` is bitwise operator, and `&=` does the operation and assigns the value back (https://www.tutorialspoint.com/python/bitwise_operators_example.htm)

```
Python 3.6.5 (default, Apr 28 2018, 22:32:18)
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> match = True
>>> match &= True
>>> print(match)
True
>>> match &= False
>>> print(match)
False
>>> match = False
>>> match &= True
>>> print(match)
False
>>> match &= False
>>> print(match)
False
```

Reply:
In short, `match` remains `True` only if all future comparisons are True. If any of them are False, `match` will become `False` and remain `False`.